bundles/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/internal/handler/capability/MeasureCapability.java

```diff
@@ -56,6 +57,10 @@ public MeasureCapability(CommonInterface handler, List<ChannelHelper> helpers) {
         measureChannelHelper.setMeasures(measures);
     }
 
+    @Override
+    protected void updateNADevice(Device newData) {
+    }
```

Comment:
I assume this is the actual fix because it will suppress some logic from a super class? Can you perhaps add a comment explaining why this empty override is necessary?

Reply:
Thanks for the explanation. To be clear: My suggestion would be to add a small code comment so that future maintainers (including yourself) will know why this empty override is important. 😉

Reply:
Done